GetMessage/userPreferenceChecker/userPreferencesCheckerFactory.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as TE from "fp-ts/TaskEither";
+import * as O from "fp-ts/Option";
+import * as b from "fp-ts/boolean";
+import { pipe } from "fp-ts/lib/function";
+
+import * as t from "io-ts";
+
+import * as semver from "semver";
+
+import { FiscalCode } from "@pagopa/io-functions-commons/dist/generated/definitions/FiscalCode";
+import { ServiceId } from "@pagopa/io-functions-commons/dist/generated/definitions/ServiceId";
+import { RetrievedProfile } from "@pagopa/io-functions-commons/dist/src/models/profile";
+import {
+  AccessReadMessageStatusEnum,
+  ServicePreference
+} from "@pagopa/io-functions-commons/dist/src/models/service_preference";
+import { Semver } from "@pagopa/ts-commons/lib/strings";
+
+/**
+ * An interface that describes all the checks over user' service prefereces
+ */
```

Comment:
dead comment?

Reply:
I forgot to move along with related interface :) updated in [5b116a0](https://github.com/pagopa/io-functions-services/pull/211/commits/5b116a0b80b4a1a5f221256a8a2049dcc8b15f60) tanks